### PR TITLE
fix empty contact bug

### PR DESCRIPTION
**Pull request by jahnaviraj**

### CODE REVIEW

File: desk/src/components/desk/contacts/ContactInfo.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
 				<div>
 					<span class="block mb-2 text-sm leading-4 text-gray-700">Profile Picture</span>
 					<div class="flex flex-row space-x-[8px] items-center">
-						<CustomAvatar :label="values?.contactName" size="2xl" :imageURL="values?.profilePicture" />
+						<CustomAvatar :label="values?.contactName" size="2xl" :imageURL="values?.profilePicture && !values.profilePicture.includes('private') && values.profilePicture" />
```
Comment (ashishnpbs) [inline desk/src/components/desk/contacts/ContactInfo.vue:24]: a &&b && c == true or false -> will imageURL be set to value or true/false, and also what if the user is looking at her own photo

File: desk/src/components/desk/settings/agents/AgentInfo.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@
             <CustomAvatar
               :label="values?.agentName"
               size="2xl"
-              :imageURL="values?.profilePicture"
+              :imageURL="values?.profilePicture && !values.profilePicture.includes('private') && values.profilePicture"
```
Comment (ashishnpbs) [inline desk/src/components/desk/settings/agents/AgentInfo.vue:44]: same as above in ContactINfo.vue

File: frappedesk/frappedesk/doctype/ticket/ticket.py [hunk 3]
```diff
@@ -53,9 +53,9 @@ def set_contact(self, email_id, save=False):
         email_id = email.utils.parseaddr(email_id)[1]
         if email_id:
             if not self.contact:
-                contact = frappe.db.get_value("Contact", {"email_id": email_id})
-                if contact:
-                    self.contact = contact
+                child_table_contact = frappe.get_list("Contact Email", fields = ['parent'], filters={'email_id': ["=", email_id]}, parent_doctype="Contact")
```
Comment (ashishnpbs) [inline frappedesk/frappedesk/doctype/ticket/ticket.py:56]: shall we do first the original and if the contact is still null do this?

**Review by ashishnpbs — COMMENTED** [review]
added few comments - will test on my local once these issues are resolved

File: desk/src/components/desk/contacts/ContactInfo.vue (hunk 1 above)
Comment (jahnaviraj) [reply desk/src/components/desk/contacts/ContactInfo.vue:24]: if && is false it returns null 
this image is only for contacts, not own profile picture. 
havent added this for own profile picture.

Comment (jahnaviraj) [reply desk/src/components/desk/contacts/ContactInfo.vue:24]: updated condition for own photo

File: desk/src/components/desk/settings/agents/AgentInfo.vue (hunk 2 above)
Comment (jahnaviraj) [reply desk/src/components/desk/settings/agents/AgentInfo.vue:44]: done same as above

File: frappedesk/frappedesk/doctype/ticket/ticket.py (hunk 3 above)
Comment (jahnaviraj) [reply frappedesk/frappedesk/doctype/ticket/ticket.py:56]: added

**Pull request merged**